### DISCUSSION TITLE
Bring back bookmark icon

### DISCUSSION THREAD

**Discussion by yedu-YK** [dailydotdev/daily#696] [Feature Requests]
I find it really annoying to bookmark articles, previously I could click on the bookmark icon now I have to go to the options menu and select it after. 

It would be so great if you could bring the icon back.

**Comment by sunilgareja** [comment]
`+ 1` 
Please add this back, it was much more convenient to bookmark posts before.

I'm no UI/UX expert but adding the bookmark icon back won't add too much to the existing template, it will still remain fairly spacious.

**Suggestion** (not sure how difficult/ how much dev time it would take)
An option to toggle it from the settings (along with an option to open article on click, instead of opening the popup)

![image](https://user-images.githubusercontent.com/19517899/188320967-ac363967-210f-469c-8ef2-93bfc148b7e5.png)

**Comment by idoshamun** [comment]
Hi,

Thanks for the suggestion this is a dup of #638.
Let's keep the discussion there.

**Comment by nimrodkra** [comment]
@yedu-YK @sunilgareja quick update: the bookmark button is back on the card

![IMG_8444](https://github.com/dailydotdev/daily/assets/41470823/bd64f777-ba63-4495-818b-0043952c0b04)
